README.md

```diff
@@ -86,6 +86,12 @@ These commands run the PHP and end-to-end test suites, respectively:
 npm run test:php
 npm run test:e2e
 ```
+You can pass extra params into the php tests by adding -- and then the [param options](https://docs.phpunit.de/en/10.4/textui.html#command-line-options)
+
+```
+npm run test:php -- --filter {test_name}
+npm run test:php -- --group {group_name}
```

Comment:
Just a thought: `--group` is also used for ticket numbers, so we can add this to the example, what do you think?


```suggestion
npm run test:php -- --group {group_name_or_ticket_number}
```